docker/default_config.ini

```diff
@@ -35,7 +35,7 @@ rpc-endpoint = 0.0.0.0:8090
 enable-stale-production = false
 
 # Percent of witnesses (0-99) that must be participating in order to produce blocks
-required-participation = false
+required-participation = 33
```

Comment:
Please comment it out instead. There's really no reason to set it explicitly.

Reply:
Will do

Reply:
Done